modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Dialogs will not exceed the window height on smaller screens #TINY-8146
 - Text alignment could not be applied to `pre` elements #TINY-7715
 
+### Improved
+- New `opaque` prop for `iframe` dialog component fixes an issue in Preview and Insert Template dialogs with dark skins #TINY-8534
```

Comment:
This is really 2 changelog entries. We should add one for the new property and one for the preview/template dialog fixes.

Also as a side note, we shouldn't be using short hand in changelogs (e.g. `prop` should be `property`).

Reply:
Ok, I was having issues fitting it into one entry but did not know if it’s fine to have to entries for one task :) Should new property be under `Improved` and dialog fix is under `Fixed` then?